docs/cloud/how-to-manage-audit-logging.md

```diff
@@ -0,0 +1,205 @@
+---
+id: how-to-manage-audit-logging
+title: How to manage Audit Logging in Temporal Cloud
+sidebar_label: Audit Logging
+description: Create, consume, and manage Audit Logs.
+toc_max_heading_level: 4
+---
+
+<!-- THIS FILE IS GENERATED. DO NOT EDIT THIS FILE DIRECTLY -->
+
+Audit Logging is a feature of <a class="tdlp" href="/cloud/index#">Temporal Cloud<span class="tdlpiw"><img src="/img/link-preview-icon.svg" alt="Link preview icon" /></span><span class="tdlpc"><span class="tdlppt">What is Temporal Cloud?</span><br /><br /><span class="tdlppd">Temporal Cloud is a managed, hosted Temporal environment that provides a platform for Temporal Applications.</span><span class="tdlplm"><br /><br /><a class="tdlplma" href="/cloud/index#">Learn more</a></span></span></a> that provides forensic access information at the account level, the user level, and the <a class="tdlp" href="/namespaces#">Namespace<span class="tdlpiw"><img src="/img/link-preview-icon.svg" alt="Link preview icon" /></span><span class="tdlpc"><span class="tdlppt">What is a Namespace?</span><br /><br /><span class="tdlppd">A Namespace is a unit of isolation within the Temporal Platform</span><span class="tdlplm"><br /><br /><a class="tdlplma" href="/namespaces#">Learn more</a></span></span></a> level.
+
+Audit Logging answers "who, when, and what" questions about Temporal Cloud resources.
+These answers can help you evaluate the security of your organization, and they can provide information that you need to satisfy audit and compliance requirements.
+
+## Supported integrations
+
+Audit Logging supports the [Amazon Kinesis](https://docs.aws.amazon.com/kinesis/) streaming-data platform.
+By using [Amazon Kinesis Data Firehose](https://docs.aws.amazon.com/firehose/latest/dev/what-is-this-service.html), you can route Temporal Audit Logs in [Amazon Simple Storage Service](https://docs.aws.amazon.com/s3/) (S3).
+We plan to release additional integrations.
+
+## Supported events
+
+The first release of Audit Logging supports Admin Operation events.
+
+### Admin Operation events
+
+Temporal logs operations at the account, user, and Namespace levels to ensure that your organization can audit and report changes to admin operations, when those changes occurred, and who made them.
+
+- Account
```

Comment:
So for this list, the left hand side of ":" is the user operation, and the right hand side of ":" is the corresponding API that called in Temporal which emitting the logs.  Do we want to add this explanation in the doc? @djmagee

Reply:
Excellent idea; I'll do so.